wemake_python_styleguide/visitors/ast/builtins.py

```diff
@@ -112,6 +112,30 @@ def _check_is_alphatbet(
                 ),
             )
 
+    def _is_exception(self, node: AnyText) -> bool:
+        """
+        Check if the string with modulo patterns is in an exceptional situation.
+
+        Basically we have some function names in which we allow strings with
+        modulo patterns because they must have them for the functions to work
+        properly.
+        """
+        exceptions = (
+            'strftime',  # For date, time, and datetime.strftime()
+            'strptime',  # For date, time, and datetime.strptime()
+            'execute',  # For psycopg2's cur.execute()
+        )
+
+        parent = nodes.get_parent(node)
```

Comment:
Can we reuse this function? https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/logic/tree/functions.py#L13

Reply:
Sure, there was some issue with the typing but I'll fix it tomorrow.